src/stage1/generated.jl

```diff
@@ -244,6 +244,10 @@ struct KwFunc{T,S}
 end
 (kw::KwFunc)(args...) = kw.kwf(args...)
 
+function ChainRulesCore.rrule(::typeof(Core.kwcall), kwargs, f, args...)
+    rrule(KwFunc(f), kwargs, f, args...)
```

Comment:
why isn't this
```suggestion
    rrule(f, args...; kwargs...)
```

Reply:
Because (I think) we want to hit this `rrule` for `KwFunc`:
https://github.com/JuliaDiff/Diffractor.jl/pull/270/files#diff-5bd76352c0319c1d2659edc9825a90b92ac204aa1c17b9b95a3880e0fd98855eR255

Reply:
I am not exactly sure why the `KwFunc` struct is needed though... maybe it could be done directly via `rrule(::typeof(Core.kwcall), ...)` directly?

Reply:
Removing the `KwFunc` seems and dispatching on `kwcall` directly seems to work, but I was afraid to remove something which I don't exactly understand
```julia
function ChainRulesCore.rrule(::typeof(Core.kwcall), kwargs, f, args...)
    r = Core.kwfunc(rrule)(kwargs, rrule, f, args...)
    if r === nothing
        return nothing
    end
    x, back = r
    x, Δ->begin
        (NoTangent(), NoTangent(), back(Δ)...)
    end
end
```

Reply:
Oh this might be the thing that is there to avoid ADing through so much of the kwarg machinery